packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -633,11 +634,16 @@ class ListTile extends StatelessWidget {
     this.trailing,
     this.isThreeLine = false,
     this.dense,
+    this.visualDensity,
     this.contentPadding,
     this.enabled = true,
     this.onTap,
     this.onLongPress,
     this.selected = false,
+    this.focusColor,
+    this.hoverColor,
+    this.focusNode,
+    this.autofocus = false,
```

Comment:
is autofocus == null allowed? If not, document and assert?

Reply:
Fixed by merging to head, where another PR added autofocus.